seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -1,474 +1,479 @@
+celo,curve,1,2023-10-17,0x4cef035b424859b5d61b1da3c2f03febd83f03083c764fa1af860bf4e26d28b3,16,0x37f750b7cc259a2f741af45294f6a16572cf5cad,118810.8502,0x765de816845861e75a25fca122bb6898b8b1282a,118999.8
+celo,curve,1,2023-10-15,0x0a524624a533492b91c1204004589225f697d37bf63af60fde884b226e8b1f26,14,0x37f750b7cc259a2f741af45294f6a16572cf5cad,10.329967,0x765de816845861e75a25fca122bb6898b8b1282a,10.34099705
+celo,curve,1,2023-10-16,0xba430c475c9ee7e8bd17c760f6a7833b0779781092033de73ca1bf3f7e86e7f8,43,0x37f750b7cc259a2f741af45294f6a16572cf5cad,5.189966,0x765de816845861e75a25fca122bb6898b8b1282a,5.197918505
+celo,curve,1,2023-10-15,0x28500eb8fe60219ee750bbea028e4ace3259aae1f350727c97311be3ac579dd4,13,0x37f750b7cc259a2f741af45294f6a16572cf5cad,432.240793,0x765de816845861e75a25fca122bb6898b8b1282a,432.6936029
+optimism,openocean,2,2023-09-04,0x1c2bc7857ce9acd8d6b018eea4b2e2affbda61e56916161b4348ef0603ff7935,23,0x350a791bfc2c21f9ed5d10980dad2e2638ffa7f6,82.11154687,0x7f5c764cbc14f9669b88837ca1490cca17c31607,500.376315
+optimism,openocean,2,2023-06-30,0x4a823ca5c73264723a1326c9d3dedd0f1b2bbbc446e7f2c71ac5c998234b8ca2,9,0x1db2466d9f5e10d7090e7152b68d62703a2245f0,14.71453526,0x7f5c764cbc14f9669b88837ca1490cca17c31607,2
+optimism,openocean,2,2022-08-03,0x13875f629d4c608d034411333bf0f0095fb6ac41c219163ed8d359805a63fa05,25,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,11.319842,0x8c6f28f2f1a3c87f0f938b96d27520d9751ec8d9,11.26571141
+base,maverick,1,2023-10-24,0xf76389f7730beb6a51a3e1881b0a9139fe8ec07d8403061fd52d5aaf1357ad00,12,0x50c5725949a6f0c72e6c4a641f24049a917db0cb,2955.014795,0xd9aaec86b65d86f6a7b5b1b0c42ffa531710b6ca,2954.059434
+zksync,maverick,1,2023-10-23,0xbe2700aac1174728513863a4f42d37aa9ac7a7d73389fa00a5abcfbd3570d5a6,4,0x2039bb4116B4EFc145Ec4f0e2eA75012D6C0f181,,0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4,
+avalanche_c,uniswap,3,2023-11-11,0x3fd0df30055592268a66399a4d958ae6aaa957cab7442c0ecc60d7257a156261,7,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,497.409883,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,34.17977416
+avalanche_c,uniswap,3,2023-11-11,0x31e1a422f13f742ba7ad4c5b2b33bf2cc6c05777cfff4062f8e9667dc0eab524,14,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,95.79678643,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,1379.142478
+avalanche_c,uniswap,3,2023-11-11,0x79cc2e2b8543ae7b5215f9f389069106061efc619fa23437b5662d81fac2d8dc,2,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,661.400762,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,46
+avalanche_c,uniswap,3,2023-11-11,0x31e1a422f13f742ba7ad4c5b2b33bf2cc6c05777cfff4062f8e9667dc0eab524,16,0x49d5c2bdffac6ce2bfdb6640f4f80f226bc10bab,0.6729956329,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,95.79678643
+avalanche_c,uniswap,3,2023-11-11,0x41b6ec0e3ba96f6c72bab7954353152ba44904bd61232f91e824c9a74495b1e4,2,0x9702230a8ea53601f5cd2dc00fdbc13d4df4a8c7,600.415091,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,41.78982883
```

Comment:
I will recover this file and add these 4 lines then, to keep this PR clean.

Reply:
what's wrong with my seed file? it will be helpful for my next PR.

Reply:
You can see your modification lost precision in numbers for other seed records.
That's why I recovered modification on other seed records, and just add relevant lines.

Reply:
I edited the csv in google sheet. That might be the reason, thanks!